### PR TITLE
add model and mako for bohrungen tiefer 500

### PR DESCRIPTION
**Pull request by nadine-piveteau**

### CODE REVIEW

**Comment by nadine-piveteau** [comment]
[try](https://mf-chsdi3.dev.bgdi.ch/ltpin_bohrungen/shorten/75117eb11e)

File: chsdi/templates/htmlpopup/bohrungen_tiefer_500.mako [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    rechtein = 'rechtein_%s' % lang
+    inhalt = 'inhalt_%s' % lang
+    auskunft = 'auskunft_%s' % lang
+
+%>
+     <tr><td class="cell-left">${_('ch.swisstopo.geologie-bohrungen_tiefer_500.name')}</td>       <td>${c['attributes']['name'] or '-'}</td></tr>
+     <tr><td class="cell-left">${_('ch.swisstopo.geologie-bohrungen_tiefer_500.auftraggeb_de')}</td>       <td>${c['attributes'][auftraggeb] or '-'}</td></tr>
+     <tr><td class="cell-left">${_('ch.swisstopo.geologie-bohrungen_tiefer_500.rechtein')}</td>       <td>${c['attributes'][rechtein] or '-'}</td></tr>
+     <tr><td class="cell-left">${_('ch.swisstopo.geologie-bohrungen_tiefer_500.inhalt')}</td>       <td>${c['attributes'][inhalt] or '-'}</td></tr>
+     <tr><td class="cell-left">${_('ch.swisstopo.geologie-bohrungen_tiefer_500.download')}</td>       <td><a href="${c['attributes']['download'] or '-'}" target="_blank">Link</a></td></tr>
```
Comment (loicgasser) [inline chsdi/templates/htmlpopup/bohrungen_tiefer_500.mako]: you shouldn't put the link at all if the download field doesn't exist.

Comment (loicgasser) [inline chsdi/templates/htmlpopup/bohrungen_tiefer_500.mako]: or at least not the anchor `<a>` tag.

Comment (nadine-piveteau) [inline chsdi/templates/htmlpopup/bohrungen_tiefer_500.mako]: Thx for the feedback! Indeed, I mispelled it... The misspelled images are now removed and I modified the .mako-file. I hope it's fine now.

**Comment by loicgasser** [comment]
this also needs rebase.

**Comment by loicgasser** [comment]
@nadine-piveteau is it ready to merge? Are we gona deploy this layer on Wednesday?

**Comment by AFoletti** [comment]
Layer is in Abnahme. Do not merge yet

**Comment by nadine-piveteau** [comment]
The changes for the tooltip, which have been requested by the clients, are now integrated in the mako-file. Thx in advance for your feedback!

**Comment by gjn** [comment]
@nadine-piveteau This is for deploy of 25th? I think it needs a merge.

**Pull request merged**